bucket/hydrus-network.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": "375",
+    "description": "A personal booru-style media tagger",
+    "homepage": "https://hydrusnetwork.github.io/hydrus/",
+    "license": "WTFPL",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/hydrusnetwork/hydrus/releases/download/v375/Hydrus.Network.375.-.Windows.-.Extract.only.zip",
+            "hash": "bc61c8ab9904a206f656a356f7686908928c3f0bdc183f0de1299287272fac7a"
+        }
+    },
+    "extract_dir": "Hydrus Network",
+    "pre_install": [
+        "'client.db', 'client.caches.db', 'client.mappings.db', 'client.master.db' | ForEach-Object {",
+        "    if (!(Test-Path \"$persist_dir\\db\\$_\")) { New-Item \"$dir\\db\\$_\" | Out-Null }",
+        "   }"
+    ],
+    "uninstaller": {
+        "script": "Get-ChildItem \"$dir\\db\\*\" -Include 'client*', 'server*' -File | Copy-Item -Destination \"$persist_dir\\db\" -Force"
+    },
+    "bin": [
+        [
+            "client.exe",
+            "hydrusclient"
+        ],
+        [
+            "server.exe",
+            "hydrusserver"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "client.exe",
+            "hydrus client"
+        ],
+        [
+            "server.exe",
+            "hydrus server"
+        ]
+    ],
+    "persist": [
```

Comment:
Why do you have this property with files listed? It is nonsense to have it when you have uninstaller/installer handler. Remove all files and keep only folders.